src/vs/workbench/contrib/accessibility/browser/accessibleView.ts

```diff
@@ -343,22 +354,32 @@ export class AccessibleView extends Disposable {
 		if (provider.id === AccessibleViewProviderId.Chat) {
 			this._register(this._codeBlockContextProviderService.registerProvider({ getCodeBlockContext: () => this.getCodeBlockContext() }, 'accessibleView'));
 		}
+		if (!('verbositySettingsKey' in provider)) {
+			this._storageService.store(`ACCESSIBLE_VIEW_SHOWN_${provider.id}`, true, StorageScope.APPLICATION, StorageTarget.USER);
+		}
 	}
 
 	previous(): void {
-		if (!this._currentProvider) {
+		if (!this._currentProvider || !('previous' in this._currentProvider)) {
```

Comment:
This is just personal preference, but I prefer having an `isIAccessibleContentProvider` function over checking for the existence of a property.

Reply:
I agree, will do 👍🏼